x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -126,11 +126,42 @@ export const MisconfigurationFindingsDetailsTable = memo(
       return getNavUrlParams({ [queryField]: name }, 'configurations', ['rule.name']);
     };
 
+    const linkWidth = 40;
+    const resultWidth = 74;
+
     const columns: Array<EuiBasicTableColumn<MisconfigurationFindingDetailFields>> = [
       {
         field: 'rule',
         name: '',
-        width: '5%',
+        width: `${linkWidth}`,
+        render: (rule: CspBenchmarkRuleMetadata, finding: MisconfigurationFindingDetailFields) => (
+          <SecuritySolutionLinkAnchor
+            deepLinkId={SecurityPageName.cloudSecurityPostureFindings}
+            path={`${getFindingsPageUrlFilteredByRuleAndResourceId(
+              rule?.id,
+              finding?.resource?.id
+            )}`}
+            target={'_blank'}
+            external={false}
+            onClick={() => {
+              uiMetricService.trackUiMetric(
+                METRIC_TYPE.CLICK,
+                NAV_TO_FINDINGS_BY_RULE_NAME_FRPOM_ENTITY_FLYOUT
+              );
+            }}
+          >
+            <EuiIcon type={'popout'} />
+          </SecuritySolutionLinkAnchor>
+        ),
+        mobileOptions: {
+          show: false,
+        },
+      },
+      {
+        field: 'rule',
+        name: i18n.translate('xpack.securitySolution.flyout.left.insights.openTitle', {
+          defaultMessage: 'Open',
```

Comment:
We don't have this title on wide screens, as the icon is kind of self-explanatory. My concern with adding this additional piece of logic is that we need to be consistent and add it to other data grids as well (vulnerability and alerts). Plus, as we use exactly the same `SecuritySolutionLinkAnchor,` it asks for moving it into a component to reuse on both mobile and non-mobile links. 
All in all, I'd personally vote for not adding additional complexity here

Reply:
I thought the elements didn't look properly aligned in the mobile view but wasn't sure to add the commit. But got it now, will drop it then :)